Makefile

```diff
@@ -157,32 +157,27 @@ all: binaries
 android-dist: genconfig android
 
 # This is to be called within the docker image.
-docker-genassets: require-npm
+genassets: require-npm
```

Comment:
The only thing that put it on the critical path for me is that for some reason, I can't package Windows when running in Docker.  If it's okay with you guys, I think I have a solution that I can show in a couple of hours.